.golangci-strict.yml

```diff
@@ -0,0 +1,176 @@
+run:
+  timeout: 10m
+linters:
+  enable:
+    - asasalint
+    - asciicheck
+    - bidichk
+    - bodyclose
+    - containedctx
+    - contextcheck
+    # - copyloopvar # requires Go 1.22
+    - cyclop
+    - decorder
+    # - depguard disable temporary until this issue is resolved: https://github.com/golangci/golangci-lint/issues/3906
```

Comment:
There was a PR that claims this one was fixed: https://github.com/golangci/golangci-lint/pull/4207
Can you check if that means this could be enabled?

Reply:
We can change the comment or remove this completely as we discussed on the recent meeting.